platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/file/FileHandler.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging.file;
+
+import com.swirlds.config.api.Configuration;
+import com.swirlds.logging.api.Level;
+import com.swirlds.logging.api.extensions.emergency.EmergencyLogger;
+import com.swirlds.logging.api.extensions.emergency.EmergencyLoggerProvider;
+import com.swirlds.logging.api.extensions.event.LogEvent;
+import com.swirlds.logging.api.extensions.handler.AbstractSyncedHandler;
+import com.swirlds.logging.api.internal.format.LineBasedFormat;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.OutputStreamWriter;
+import java.nio.file.Path;
+import java.util.Objects;
+
+public class FileHandler extends AbstractSyncedHandler {
+
+    private static final String FILE_NAME_PROPERTY = "%s.file";
+    private static final String BUFFER_SIZE_PROPERTY = "%s.bufferSize";
+    private static final String APPEND_PROPERTY = "%s.append";
+    private static final String DEFAULT_FILE_NAME = "swirlds-log.log";
+
+    /**
+     * The emergency logger.
+     */
+    private static final EmergencyLogger EMERGENCY_LOGGER = EmergencyLoggerProvider.getEmergencyLogger();
```

Comment:
This seems like it would (eventually) be common across multiple handlers.  Would it make sense to move this static value to a protected attribute in the `AbstractLogHandler` class?

Reply:
Good catch, moved